jaeger-core/src/main/java/io/jaegertracing/Configuration.java

```diff
@@ -181,10 +181,6 @@
   private Map<String, String> tracerTags;
   private boolean useTraceId128Bit;
 
-  /**
-   * lazy singleton JaegerTracer initialized in getTracer() method.
-   */
-  private JaegerTracer tracer;
```

Comment:
This will have to stay here as people are caching `Configuration.getTracer()`.

 We just don't want to cache other parts related to tracer builder.

Reply:
agreed